packages/service-library/src/servicelib/redis.py

```diff
@@ -87,6 +108,21 @@ async def ping(self) -> bool:
             return True
         return False
 
+    async def _check_health(self) -> None:
+        self._is_healthy = await self.ping()
+
+    @property
+    def is_healthy(self) -> bool:
+        """Provides the status of Redis.
+        If redis becomes available, after being not available,
+        it will once more return ``True``
```

Comment:
what does this sentence mean? it is the last heatlth check result right?

Reply:
Yes, let me rewrite it like that.